ocean_lib/models/v4/fixed_rate_exchange.py

```diff
@@ -3,18 +3,42 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 from typing import List
+from enum import IntEnum
 
 from enforce_typing import enforce_types
 
 from ocean_lib.web3_internal.contract_base import ContractBase
 from ocean_lib.web3_internal.wallet import Wallet
 
 
+class FixedRateExchangeV4Details(IntEnum):
```

Comment:
For me looks like a good approach to use enums classes here. @calina-c what do you think?

Reply:
I am not a fan of Enums since they make the usage of the class more unintuitive, but my feelings are not particularly strong on this.